agreements/ddl.sql

```diff
@@ -57,6 +57,25 @@ CREATE TABLE lot_sectors (
 
 CREATE INDEX LOT_SECTORS_IDX1 on LOT_SECTORS (lot_id,sector_code);
 
+CREATE TABLE route_to_market (
+  route_to_market_name              VARCHAR(50) PRIMARY KEY,
+  route_to_market_description       VARCHAR(2000) NOT NULL
+);
+
+CREATE TABLE lot_route_to_market (

```

Comment:
Cannot have 2 primary keys defined - syntax needs to be:

```
CREATE TABLE lot_route_to_market (
  lot_id                            INTEGER,
  route_to_market_name              VARCHAR(50),
  start_date                        DATE NOT NULL,
  end_date                          DATE NOT NULL,
  location                          VARCHAR(20), -- For example, Regional or National  
  lot_minimum_value                 NUMERIC(18,4),         
  lot_contract_minimum_weeks        SMALLINT, 
  lot_contract_maximum_weeks        SMALLINT,
  lot_procurement_minimum_quantity  SMALLINT,
  lot_procurement_maximum_quantity  SMALLINT,
  PRIMARY KEY (lot_id, route_to_market_name)
);
```

Same for `lot_sectors`:

```
CREATE TABLE lot_sectors (
  lot_id                            INTEGER,
  sector_code                       VARCHAR(2),
  PRIMARY KEY (lot_id, sector_code)
);
```

Reply:
Will make those changes and add the drop table statement.